HHS/OPRE-OPS
# feat: ✨ adds CAN Cards on Spending Tab

## What changed

- adds CAN Cars on spending tab
- agreement data is hard-coded for now

## Issue

#2672 #3046

## How to test

1. goto cans list
2. select 2021 from fy filter
3. select CAN 4426
4. goto spending tab
5. select 2043 from fy filter
6. see cards update

## Screenshots

<img width="1009" alt="image" src="https://github.com/user-attachments/assets/a6220dba-d066-41f7-90b0-b1102ff0b5e7">

## Definition of Done Checklist
